components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss

```diff
@@ -24,6 +29,50 @@ chef-icon.SKIPPED {
   color: $chef-dark-grey;
 }
 
-chef-tooltip {
-  width: 400px;
+.run-info-tooltip {
+  border-left: $spacing-03 solid;
+  border-right: $spacing-03 solid transparent;
+  padding-right: $spacing-03;
```

Comment:
I think you can just delete these - looks like together these cancel eachother out